reference/base/lapack_bindings.hpp

```diff
@@ -0,0 +1,225 @@
+// SPDX-FileCopyrightText: 2025 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifndef GKO_REFERENCE_BASE_LAPACK_BINDINGS_HPP_
+#define GKO_REFERENCE_BASE_LAPACK_BINDINGS_HPP_
+
+#include <ginkgo/core/base/exception_helpers.hpp>
+#include <ginkgo/core/base/math.hpp>
+#include <ginkgo/core/base/types.hpp>
+
+#if GKO_HAVE_LAPACK
+
+
+extern "C" {
+void ssygvd(const std::int32_t* itype, const char* jobz, const char* uplo,
+            const std::int32_t* n, float* A, const std::int32_t* lda, float* B,
+            const std::int32_t* ldb, float* w, float* work,
+            const std::int32_t* lwork, std::int32_t* iwork,
+            const std::int32_t* liwork, std::int32_t* info);
+
+void dsygvd(const std::int32_t* itype, const char* jobz, const char* uplo,
+            const std::int32_t* n, double* A, const std::int32_t* lda,
+            double* B, const std::int32_t* ldb, double* w, double* work,
+            const std::int32_t* lwork, std::int32_t* iwork,
+            const std::int32_t* liwork, std::int32_t* info);
+
+void chegvd(const std::int32_t* itype, const char* jobz, const char* uplo,
+            const std::int32_t* n, std::complex<float>* A,
+            const std::int32_t* lda, std::complex<float>* B,
+            const std::int32_t* ldb, float* w, std::complex<float>* work,
+            std::int32_t* lwork, float* rwork, std::int32_t* lrwork,
+            std::int32_t* iwork, std::int32_t* liwork, std::int32_t* info);
+
+void zhegvd(const std::int32_t* itype, const char* jobz, const char* uplo,
+            const std::int32_t* n, std::complex<double>* A,
+            const std::int32_t* lda, std::complex<double>* B,
+            const std::int32_t* ldb, double* w, std::complex<double>* work,
+            std::int32_t* lwork, double* rwork, std::int32_t* lrwork,
+            std::int32_t* iwork, std::int32_t* liwork, std::int32_t* info);
+}
```

Comment:
```suggestion


}

```

Reply:
When I added the space before the closing brace here, `clang-format` removed it again during the pre-commit hook. I added the other spaces.